docs/website/Makefile

```diff
@@ -1,21 +1,23 @@
 TOP=../../..
-WEBSITE_DOCS ?= $(TOP)/documentation
+WEBSITE_DOCS ?= $(TOP)/xamarin-docs-pr
```

Comment:
@spouliot these docs changes were made in d15-7 (PR #3771), and are here because we branched the `xcode10` branch from `d15-7`. 

This means `d15-8` does not have these changes (nor does master (yet)). Was that intentional?